src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -781,6 +781,89 @@ def test_containerapp_dev_service_binding_e2e(self, resource_group):
 
         self.cmd(f'containerapp env delete -g {resource_group} -n {env_name} --yes')
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="eastus2")
+    def test_containerapp_addon_binding_e2e(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        env_name = self.create_random_name(prefix='containerapp-env', length=24)
+        ca_name = self.create_random_name(prefix='containerapp', length=24)
+        image = "mcr.microsoft.com/azuredocs/containerapps-helloworld:latest"
+        redis_ca_name = 'redis'
+        postgres_ca_name = 'postgres'
+        kafka_ca_name = 'kafka'
+        mariadb_ca_name = 'mariadb'
+        qdrant_ca_name = "qdrant"
+        ADDON_LIST = ["redis", "postgres", "kafka", "mariadb", "qdrant"]
+        create_containerapp_env(self, env_name, resource_group)
+
+        self.cmd('containerapp addon redis create -g {} -n {} --environment {}'.format(
```

Comment:
please check the `provisiongState `for result. Same as blow.

Reply:
the provisioningState of all addons is checked in the for loop